articles/machine-learning/how-to-machine-learning-interpretability-aml.md

```diff
@@ -128,7 +128,7 @@ Refer to the following example to help you get the aggregate (global) feature im
 global_explanation = explainer.explain_global(x_train)
 
 # if you used the PFIExplainer in the previous step, use the next line of code instead
-# global_explanation = explainer.explain_global(x_train, true_labels=y_test)
+# global_explanation = explainer.explain_global(x_test, true_labels=y_test)
```

Comment:
Please keep `x_train` the same and use `y_train` instead of `y_test` in your proposed change in this PR - That way it will be more consistent with the code used above this explainer.

Reply:
Just added the new commit. Thanks!